src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableOrchestrationContext.cs

```diff
@@ -292,7 +292,6 @@ private DurableHttpRequest CreateLocationPollRequest(DurableHttpRequest durableH
                 method: HttpMethod.Get,
                 uri: new Uri(locationUri),
                 headers: durableHttpRequest.Headers,
```

Comment:
Maybe this can be a separate PR, but I wonder if we need to reduce the set of headers we send to the polling endpoint. I recall there were some issues with this that Connor had to fix at one point.

Reply:
Yeah. I think the assumption we made when designing this feature was that the request would be identical to the initial request, just with a different URI.

I think it has been made clear that in practice this can lead to some problems. I'm not sure what the proper default polling request looks like honestly. I'm a bit afraid to make to many drastic changes do to it technically being a breaking change, but I think we can spin this as more of a bug fix. Once we tackle #1183, we will have given everyone a wide enough escape valve that if they don't like our defaults, they can use their own thankfully.